pkg/daemon/discover/discover.go

```diff
@@ -46,6 +50,7 @@ var (
 	lastDevice      string
 	cmName          string
 	cm              *v1.ConfigMap
+	udevEventPeriod = time.Duration(2) * time.Second
```

Comment:
Does 2 seconds capture all the noisy events from what you've seen? Increasing a few more seconds if needed would be fine if it reduces any more noise.

Reply:
I'll increase it a bit. It was sufficient to catch basic noise, but the harder noise to capture is adding / removing multiple disks at a larger (e.g. 30 seconds) timescales. That probably needs to be handled at a higher level in which we can detect if an orchestration event covers a period of time that includes or excludes a hotplug event.

Reply:
Yeah we probably want to keep it under 5 seconds. If there are multiple orchestrations triggered that's fine, and  the operator will now handle batching them up together if multiple orchestrations a requested before the previous one is done.